ceimport/sites/musicbrainz.py

```diff
@@ -49,20 +86,26 @@ def load_person_from_musicbrainz(artist_mbid):
     born = died = None
     if lifespan:
         born = lifespan.get('begin')
+        if born:
+            if [x for x in born.split("-") if not x.isdigit()]:
```

Comment:
I found a case where the date was like:
xxxx-12-03

So it failed when importing the date because it did not recognize the year

Reply:
as far as I'm aware musicbrainz doesn't allow non-numerical values for dates. It'd be good if we could find the exact case where this happens and add a comment that describes what it's for.

Reply:
Found :) 
Born: ????-01-02
https://musicbrainz.org/artist/e1e86dcd-15a3-4df2-9ecf-172d1ef2c58f

Reply:
ah, this makes sense. thanks for finding it. This will happen in a case where there is a more specific value, but the less specific value is missing (if there is mm-dd but no year, it will show ????, but if there is year and no date it'll just show yyyy).
In this case let's improve the test by doing 
`if all([x for x in born.split("-") if x.isdigit()]:`
which is a little clearer